Samples/UniversalDecoder/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "universaldecoder",
```

Comment:
Do you typically commit the package-lock.json in node projects?

Reply:
Yes it is intended to be commited: https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md